test/e2e/pkg/client/pod.go

```diff
@@ -32,9 +31,9 @@ func (w *K8sClient) WaitForPodRunningByLabel(label string) (deployed bool, err e
 	for {
 		select {
 		case <-timeout:
-			return false, errors.New("timed out")
+			return false, err
 		case <-tick:
-			err := w.WaitForRunningPodBySelector(config.Namespace, label, 3*time.Minute)
+			err := w.WaitForRunningPodBySelector(config.Namespace, label, time.Minute)
```

Comment:
To avoid redefining err here:
```suggestion
			err = w.WaitForRunningPodBySelector(config.Namespace, label, time.Minute)
```